src/components/Tooltip/Tooltip.cy.tsx

```diff
@@ -191,4 +198,111 @@ describe('Tooltip Component', () => {
         cy.get(TOOLTIP_ID).should('exist');
         cy.get(TOOLTIP_ID).should('have.class', 'tw-hidden');
     });
```

Comment:
I think there should be a test added for the content prop itself.

Reply:
The first test case has only the `content` prop in it, so it is already there. I will update the title to `should render a tooltip with content`.
BTW, `content` is the only required prop.